pkg/route/api/validation/validation.go

```diff
@@ -67,14 +67,14 @@ func ValidateRouteStatusUpdate(route *routeapi.Route, older *routeapi.Route) fie
 	return allErrs
```

Comment:
Add a comment to this method indicating that it can't call `ValidateRouteUpdate` or we'll risk breaking